libs/application/template-api-modules/src/lib/modules/templates/old-age-pension/old-age-pension-utils.ts

```diff
@@ -0,0 +1,110 @@
+import { Application, YES } from '@island.is/application/types'
+import {
+  ApplicationType,
+  getApplicationAnswers,
+} from '@island.is/application/templates/old-age-pension'
+import { parse } from 'date-fns'
+import { Child, OldAgePension } from '@island.is/clients/social-insurance-administration'
+
+export const transformApplicationToOldAgePensionDTO = (
+  application: Application,
+  uploads: any,
```

Comment:
Er ekki hægt að nýta "Uploads" úr `import { Uploads } from '@island.is/clients/social-insurance-administration'` hérna?
```suggestion
  uploads: Uploads,
```